tool/release/createnotes/Main.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.vaticle.dependencies.tool.release.createnotes
+
+import java.nio.file.Paths
+import kotlin.io.path.notExists
+
+fun main(args: Array<String>) {
```

Comment:
This is the entry point of the program. Given a repo, version, commit and a template file:
- it will collect all the commits between the commit that is to be released to the last released commit
- for each collected commit, write the title and description into the release note